libqtile/utils.py

```diff
@@ -25,10 +25,21 @@
 import traceback
 import warnings
 from collections.abc import Sequence
+from random import randint
 from shutil import which
 
 from libqtile.log_utils import logger
 
+_can_notify = False
+try:
+    import gi
+    gi.require_version("Notify", "0.7")  # type: ignore
+    from gi.repository import Notify  # type: ignore
+    Notify.init("Qtile")
+    _can_notify = True
+except Exception as e:
```

Comment:
Can we use ImportError here if we only expect import errors?

Reply:
yep done.